### PR TITLE
add missing packages

### PR DESCRIPTION
**Pull request by men1n2**

### CODE REVIEW

**Comment by popey** [comment]
Hi @men1n2  - many thanks for the update.
I think the `nextcloud` snap is actually the server, and not the desktop client, so I don't think we should be translating that one. I know there used to be a nextcloud desktop snap in the store, but I think it was either made private or unpublished.

**Comment by philclifford** [comment]
> nextcloud desktop snap in the store, but I think it was either made private or unpublished.

```
snap list|grep nextcloud
nextcloud-client                 2.3.3+gitab40efe            10     latest/stable     nextcloud*          
```
I had this lying around - binning it now (thanks for the prompt !)

File: applist.csv [hunk 1]
```diff
@@ -45,8 +46,10 @@ goland,com.jetbrains.GoLand
 granatier,org.kde.granatier
 gwenview,org.kde.gwenview
 inkscape,org.inkscape.Inkscape
+insomnia,rest.insomnia.Insomnia
 intellij-idea-community,com.jetbrains.IntelliJ-IDEA-Community
 intellij-idea-ultimate,com.jetbrains.IntelliJ-IDEA-Ultimate
+joplin,net.cozic.joplin_desktop
```
Comment (philclifford) [inline applist.csv:52]: I think this is replacing the terminal version with the desktop app: the desktop app snap that matches this flatpak seems to me to be
joplin-desktop

Comment (men1n2) [reply applist.csv:52]: indeed, I got it installed and didn't realise that it was a cli version
will fix, thanks

**Comment by men1n2** [comment]
> Hi @men1n2 - many thanks for the update. I think the `nextcloud` snap is actually the server, and not the desktop client, so I don't think we should be translating that one. I know there used to be a nextcloud desktop snap in the store, but I think it was either made private or unpublished.

Indeed, the desktop snap is no more available, I will remove it

**Comment by men1n2** [comment]
@popey I fixed the two packages mentioned in the comments and resolved the conflict, the MR should be good now

**Comment by popey** [comment]
Looks good! Thank you @men1n2

**Pull request merged**